anchor/fuzz/fuzz_targets/setup.rs

```diff
@@ -0,0 +1,155 @@
+#![allow(dead_code)]
+use std::{
+    collections::VecDeque,
+    path::Path,
+    sync::{Arc, LazyLock, Mutex},
+    time::Duration,
+};
+
+use database::NetworkDatabase;
+use message_receiver::{NetworkMessageReceiver, Outcome};
+use message_sender::NetworkMessageSender;
+use message_validator::Validator;
+use openssl::rsa::Rsa;
+use qbft::{
+    Config, ConfigBuilder, DefaultLeaderFunction, InstanceHeight, Qbft, UnsignedWrappedQbftMessage,
+};
+use qbft_manager::QbftManager;
+use signature_collector::SignatureCollectorManager;
+use slot_clock::{ManualSlotClock, SlotClock};
+use ssv_types::{consensus::BeaconVote, domain_type::DomainType, msgid::MessageId, OperatorId};
+use subnet_tracker::SubnetId;
+use task_executor::TaskExecutor;
+use tokio::sync::mpsc;
+use types::{Hash256, Slot};
+
+type MessageQueue = Arc<Mutex<VecDeque<(OperatorId, UnsignedWrappedQbftMessage)>>>;
+type QbftSendFn = Box<dyn FnMut(UnsignedWrappedQbftMessage) + Send + Sync>;
+
+pub static VALIDATOR: LazyLock<Arc<Validator<ManualSlotClock>>> =
+    LazyLock::new(setup_test_message_validator);
+
+pub static RUNTIME: LazyLock<tokio::runtime::Runtime> =
+    LazyLock::new(|| tokio::runtime::Runtime::new().unwrap());
+
+pub static RECEIVER: LazyLock<Arc<NetworkMessageReceiver<ManualSlotClock>>> =
+    LazyLock::new(setup_test_message_receiver);
+
+pub static QBFT: LazyLock<Arc<Mutex<Qbft<DefaultLeaderFunction, BeaconVote, QbftSendFn>>>> =
+    LazyLock::new(|| {
+        let msg_queue: MessageQueue = Arc::new(Mutex::new(VecDeque::new()));
+        let id = OperatorId::from(1);
+
+        let send_message: QbftSendFn = Box::new(move |message| {
+            let mut queue = msg_queue.lock().unwrap();
+            queue.push_back((id, message));
+        });
+
+        Arc::new(Mutex::new(setup_qbft_instance(send_message)))
+    });
+
+// Setup a new Qbft instance
+pub fn setup_qbft_instance(
+    send_message: QbftSendFn,
+) -> Qbft<DefaultLeaderFunction, BeaconVote, QbftSendFn> {
+    let config: Config<DefaultLeaderFunction> = ConfigBuilder::new(
+        1.into(),
+        InstanceHeight::default(),
+        (1..=4).map(OperatorId::from).collect(),
+    )
+    .build()
+    .unwrap();
+
+    let data = BeaconVote {
+        block_root: Hash256::random(),
+        source: types::Checkpoint::default(),
+        target: types::Checkpoint::default(),
+    };
+
+    Qbft::new(config, data, MessageId::from([0; 56]), send_message)
+}
+
+// Sets up a real Validator for fuzzing
+pub fn setup_test_message_validator() -> Arc<Validator<ManualSlotClock>> {
+    let slot_clock = ManualSlotClock::new(
```

Comment:
Likely want to reset the slot clock (and any other state) between fuzz invocations.
Want each execution to be isolated from the others